### ISSUE TITLE
Upgrade fails due to topics, then due to OPF

### ISSUE THREAD

**Issue by instantflorian** [WBCE/WebsiteBaker_CommunityEdition#186]
1) Fatal error: Uncaught Error: Call to undefined function mysql_error() in /.../modules/topics/upgrade.php:41 Stack trace: #0 /.../upgrade-script.php(407): require_once() #1 {main} thrown in /../modules/topics/upgrade.php on line 41
=> there are misc. occurances of mysql_error in upgrade.php and /scripts/upgrade.inc.php which have to be removed

2) 
Warning: opf_is_registered(): Filter not registred in /.../modules/outputfilter_dashboard/functions.php on line 673

Warning: opf_is_registered(): Filter not registred in /.../modules/outputfilter_dashboard/functions.php on line 673

Fatal error: Uncaught Error: Call to undefined function opf_is_registed() in /.../modules/mod_opf_short_url/install.php:38 Stack trace: #0 /.../modules/outputfilter_dashboard/install.php(115): require() #1 /.../upgrade-script.php(417): require_once('/var/www/web913...') #2 {main} thrown in /.../modules/mod_opf_short_url/install.php on line 38

No idea how to fix

**Comment by instantflorian** [comment]
...and after commenting out the lines regarding opf the script crashes with

Warning: Cannot modify header information - headers already sent by (output started at /.../upgrade-script.php:317) in /.../framework/globalExceptionHandler.php on line 4

just after the headline
Step 3 : Upgrade module 'news' to version 3.5.6

**Comment by NorHei** [comment]
` News, haben wir doch gar keins drinn ?

**Comment by NorHei** [comment]
Wo kann ich das schauen ?

**Comment by NorHei** [comment]
This should be repaired long time ago ...

**Comment by NorHei** [comment]
Or can anyone reproduce this ?

**Comment by mrbaseman** [comment]
At least the opf_is_registed was a typo in the function name, fixed in branch 1.2.x on Jan, 17th or so... (in master there might still be an old version)

**Comment by instantflorian** [comment]
Did a test update today, no fatal errors anymore, just some warnings.

**Comment by mrbaseman** [comment]
Just for completeness: According to my records I have fixed the "warnings on line 673" on Jan, 22th. And about the "Warning: Cannot modify header information": In a private communication with NorHei I have seen that this was related to those other warnings, which would trigger a redirect in  framework/globalExceptionHandler.php during the run of the upgrade script after the output has already started. Since the warnings on line 673 have been fixed, this issue was also resolved.
